frontend/src/views/account-mgmt/ManageUsersView.vue

```diff
@@ -117,7 +138,7 @@ export default {
       userToUpdate: {},
       userToDeactivate: {},
       headersUsers: [
-        { title: '', key: 'data-table-expand', width: '87px' },
+        { title: '', key: 'dataTableExpand', width: '30px' },
```

Comment:
I think this might be a little small as **hide detail** ends up on two lines at desktop resolution.

Reply:
width is reverted back to 87.